cannot use file (type *io.Reader) as type io.Reader in argument to io.Copy

Hey @JonasFranzDEV, I get the following compilation error using the latest [commit](https://github.com/go-gitea/go-sdk/commit/c026786979c62f631658f365fd26e277cd90bfa4):

```
# bitbucket.org/gridarrow/gridarrow-manager/vendor/code.gitea.io/sdk/gitea
vendor/code.gitea.io/sdk/gitea/attachment.go:11:2: imported and not used: "io/ioutil"
vendor/code.gitea.io/sdk/gitea/attachment.go:58:21: cannot use file (type *io.Reader) as type io.Reader in argument to io.Copy:
	*io.Reader is pointer to interface, not interface
```

This is with the following glide dependency:
```
- package: code.gitea.io/sdk
  subpackages:
  - gitea
```

Previous commit works fine for me. Not a go expert, so perhaps it's my bad somehow...

--- a/gitea/attachment.go
+++ b/gitea/attachment.go
@@ -45,7 +45,7 @@ func (c *Client) GetReleaseAttachment(user, repo string, release int64, id int64
 }
 
 // CreateReleaseAttachment creates an attachment for the given release
-func (c *Client) CreateReleaseAttachment(user, repo string, release int64, file *io.Reader, filename string) (*Attachment, error) {
+func (c *Client) CreateReleaseAttachment(user, repo string, release int64, file io.Reader, filename string) (*Attachment, error) {
 	// Write file to body
 	body := new(bytes.Buffer)
 	writer := multipart.NewWriter(body)
@@ -54,7 +54,7 @@ func (c *Client) CreateReleaseAttachment(user, repo string, release int64, file 
 		return nil, err
 	}
 
-	if _, err = io.Copy(part, *file); err != nil {
+	if _, err = io.Copy(part, file); err != nil {
 		return nil, err
 	}
 	if err = writer.Close(); err != nil {

--- a/gitea/attachment.go
+++ b/gitea/attachment.go
@@ -8,7 +8,6 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
-	"io/ioutil"
 	"mime/multipart"
 	"net/http"
 	"time"
@@ -55,7 +54,7 @@ func (c *Client) CreateReleaseAttachment(user, repo string, release int64, file 
 		return nil, err
 	}
 
-	if _, err = io.Copy(part, file); err != nil {
+	if _, err = io.Copy(part, *file); err != nil {
 		return nil, err
 	}
 	if err = writer.Close(); err != nil {

